Detections/MultipleDataSources/NetworkEndpointCorrelation.yaml

```diff
@@ -1,4 +1,4 @@
-﻿id: 01f64465-b1ef-41ea-a7f5-31553a11ad43
```

Comment:
Build failed, need you to review the change you did here.

Reply:
not review...remove the change you did to the GUID.

Reply:
ahh ok thank you will do